### PR TITLE
UI Plugin: expand wizard width, add back buttons

### PR DESCRIPTION
**Pull request by miclettej**
## What this PR does / why we need it

Expands width of management cluster and unmanaged cluster wizards to full screen

Adds back arrow buttons to title of each management cluster provider wizard

Adds icon to unmanaged cluster wizard title

Fixes some nitpicks on title wording in unmanaged clusters

## Which issue(s) this PR fixes

Fixes: #4487

### CODE REVIEW

File: cli/cmd/plugin/ui/web/tanzu-ui/src/views/management-cluster/azure/AzureManagementCluster.tsx [hunk 1]
```diff
@@ -1,31 +1,44 @@
...
+import { NavRoutes } from '../../../shared/constants/NavRoutes.constants';
 
 function AzureManagementCluster() {
+    const navigate = useNavigate();
+
     return (
         <AzureProvider>
             <div cds-layout="grid col:12">
-                <div cds-layout="col:8">
+                <div cds-layout="col:12">
```
Comment (raymondz1) [inline cli/cmd/plugin/ui/web/tanzu-ui/src/views/management-cluster/azure/AzureManagementCluster.tsx:22]: we have col: 12 in parent container, so I think we may not need this `div`

File: cli/cmd/plugin/ui/web/tanzu-ui/src/views/management-cluster/docker/DockerManagementCluster.tsx [hunk 2]
```diff
@@ -1,30 +1,43 @@
...
+import { NavRoutes } from '../../../shared/constants/NavRoutes.constants';
 
 function DockerManagementCluster() {
+    const navigate = useNavigate();
+
     return (
         <DockerProvider>
             <div cds-layout="grid col:12">
-                <div cds-layout="col:8">
+                <div cds-layout="col:12">
```
Comment (raymondz1) [inline cli/cmd/plugin/ui/web/tanzu-ui/src/views/management-cluster/docker/DockerManagementCluster.tsx:22]: same here

File: cli/cmd/plugin/ui/web/tanzu-ui/src/views/management-cluster/vsphere/VsphereManagementCluster.tsx [hunk 3]
```diff
@@ -1,29 +1,43 @@
...
+import { NavRoutes } from '../../../shared/constants/NavRoutes.constants';
 
 function VsphereManagementCluster() {
+    const navigate = useNavigate();
+
     return (
         <VsphereProvider>
             <div className="vsphere-management-container" cds-layout="grid col:12">
-                <div cds-layout="col:8">
+                <div cds-layout="col:12">
```
Comment (raymondz1) [inline cli/cmd/plugin/ui/web/tanzu-ui/src/views/management-cluster/vsphere/VsphereManagementCluster.tsx:21]: same here

File: cli/cmd/plugin/ui/web/tanzu-ui/src/views/unmanaged-cluster/UnmanagedClusterWizard.tsx [hunk 4]
```diff
@@ -24,13 +24,14 @@ function UnmanagedClusterWizard() {
     return (
         <UmcProvider>
             <div cds-layout="grid col:12">
-                <div cds-layout="col:8">
+                <div cds-layout="col:12">
```
Comment (raymondz1) [inline cli/cmd/plugin/ui/web/tanzu-ui/src/views/unmanaged-cluster/UnmanagedClusterWizard.tsx:27]: and here

**Review by raymondz1 — APPROVED** [review]
LGTM

**Review by swalner-vmware — APPROVED** [review]

**Pull request merged**